elasticsearch/connection_pool.py

```diff
@@ -227,6 +227,14 @@ def get_connection(self):
         # only one connection, no need for a selector
         return connections[0]
 
+    def close(self):
+        """
+        Explicitly closes connections
+        """
+        for conn in self.connections:
```

Comment:
we should probably try and close the dead connections too. Iterating over `orig_connections` instead should do it